rclrs/src/node/mod.rs

```diff
@@ -212,4 +212,38 @@ impl Node {
             .filter_map(Weak::upgrade)
             .collect()
     }
+
+    /// Returns the ROS domain ID that the node is using.
+    ///    
+    /// returned values depend on ROS_DOMAIN_ID environment variable.   
+    ///
+    /// # Example
+    /// ```
+    /// use rclrs::{Context, RclReturnCode};
```

Comment:
This line is always hidden in the other examples, using `#`. Would be nice to do this here too for consistency.

Reply:
Do you mean using `#` before `use` statement?

Reply:
You mean use `#` leading `use` statement?